web-services/security/src/main/java/datawave/security/user/UserOperationsBean.java

```diff
@@ -45,6 +46,7 @@ public class UserOperationsBean {
     private CredentialsCacheBean credentialsCache;
     
     @Inject
+    @SpringBean(name = "ResponseObjectFactory")
```

Comment:
Likewise is `@SpringBean` required here as well?

Reply:
yes we need a spring bean here. This is an EJB that is created via CDI, and the response object factory is a spring created object.